sqlglot/dialects/clickhouse.py

```diff
@@ -453,6 +453,15 @@ def _parse_create(self) -> exp.Create | exp.Command:
 
             return create
 
+        def _parse_drop(self, exists: bool = False) -> exp.Drop | exp.Command:
+            drop = super()._parse_drop(exists)
+
+            # DATABASE in ClickHouse is the same as SCHEMA in other dialects
+            if isinstance(drop, exp.Drop) and drop.args["kind"] == "DATABASE":
+                drop.args["kind"] = "SCHEMA"
```

Comment:
```suggestion
            if isinstance(drop, exp.Drop) and drop.kind == "DATABASE":
                drop.set("kind", "SCHEMA")
```

Reply:
That's how I originally did it (mimicking your create PR), but:

```
  File "/Users/trey/tobiko/sqlglot/sqlglot/dialects/clickhouse.py", line 460, in _parse_drop
    if isinstance(drop, exp.Drop) and drop.kind == "DATABASE":
                                      ^^^^^^^^^
AttributeError: 'Drop' object has no attribute 'kind'
```

Reply:
Ah I see. Let's add that property to `Drop` too then, I think it makes sense to reflect what we do for `Create`.